Paintroid/src/main/java/org/catrobat/paintroid/ui/MainActivityNavigator.java

```diff
@@ -338,13 +337,13 @@ public void showLoadErrorDialog() {
 
 	@Override
 	public void showRequestPermissionRationaleDialog(PermissionInfoDialog.PermissionType permissionType, String[] permissions, int requestCode) {
-		AppCompatDialogFragment dialog = PermissionInfoDialog.newInstance(permissionType, permissions, requestCode);
+		AppCompatDialogFragment dialog = PermissionInfoDialog.Companion.newInstance(permissionType, permissions, requestCode);
```

Comment:
I think you can remove `.Companion` 🤔, please make similar changes at other places too